GlotPress/GlotPress-WP
# Local REST API: Implement Saving

## What?
This removes the need for the https://github.com/akirk/gp-translation-extended-api API for translation saving to work.

## Why?
The functionality should be in GlotPress.

## How?
This starts implementing the REST API which we want later to replace the standard GlotPress API.

## Testing Instructions
1. Edit a translation or submit a new one.
2. Observe an XHR sent with a positive response.
3. Reload the page to see that it sticks.

## Screenshots or screencast
